components/footer/footer.jsx

```diff
@@ -1,42 +1,86 @@
-import React from 'react';
-import { MofoFooter } from 'mofo-ui';
+import React from "react";
 
-const Footer = () => {
-  const FOOTER_LINKS = [
-    {
-      iconType: `github`,
-      link: `https://github.com/mozilla/network-pulse`,
-      text: `Github`
-    },
-    {
-      iconType: `legal`,
-      link: `https://www.mozilla.org/about/legal/terms/mozilla/`,
-      text: `Legal`
-    },
-    {
-      iconType: `cc-license`,
-      link: `https://creativecommons.org/licenses/by/4.0/`,
-      text: `License`
-    },
-    {
-      iconType: `privacy`,
-      link: `https://www.mozilla.org/privacy/websites/`,
-      text: `Privacy`
-    }
-  ];
-
-  const ORGS = [
-    {
-      name: `mozilla`,
-      link: `https://mozilla.org`,
-      description: (<p className="dark-theme body-small">Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>.</p>),
-      className: `mozilla`
-    }
-  ];
-
-  return (
-    <MofoFooter footerLinks={FOOTER_LINKS} orgs={ORGS} />
-  );
-};
-
-export default Footer;
+export default class MofoFooter extends React.Component {
+  renderSocialLinks() {
+    return (
+      <ul className="d-flex pl-0">
+        <li className="mr-2">
+          <a
+            className="twitter small d-inline-block"
+            href="https://twitter.com/mozilla">
+            <span className="sr-only">Twitter</span>
+          </a>
+        </li>
+        <li className="mr-2">
+          <a
+            className="instagram small d-inline-block"
+            href="https://www.instagram.com/mozilla/">
+            <span className="sr-only">Instagram</span>
+          </a>
+        </li>
+        <li className="mr-2">
+          <a
+            className="github small d-inline-block"
+            href="https://www.github.com/mozilla/foundation.mozilla.org/">
+            <span className="sr-only">Github</span>
+          </a>
+        </li>
+      </ul>
+    );
+  }
+  renderFooterLinks() {
+    return (
+      <ul className="link-list list-unstyled mb-0">
+        <li className="col-auto mb-2" key="Donate">
+          <a className="footer-link-donate" href="https://donate.mozilla.org/?utm_source=foundation.mozilla.org&amp;utm_medium=referral&amp;utm_content=footer">Donate</a>
```

Comment:
@xmatthewx any idea what utm code etc we should use for the Donate line on Pulse footer?

Reply:
@youriwims the link included here is specifically for the footer Donate link on Foundation site. Can you update the link after @xmatthewx replies?

Reply:
sounds good 👍

Reply:
Got it ➡️ updated link ➡️ https://github.com/mozilla/network-pulse/issues/1036#issuecomment-492395064
Code update: https://github.com/mozilla/network-pulse/pull/1054/files#diff-aed0ade64de8902416d11ddfa47d85d0R35